### ISSUE TITLE
Ticket 699. disable autocomplete email.

### ISSUE THREAD

**Issue by ivanromakh** [Scholar-6/brillder#4612]

**Comment by s-steddy** [comment]
Final fix, I hope: If I'm in manage account it looks like a pin is still autofilled, and scroll over field turns it blue, obscuring text. Blue goes away if you delete the digits though.

<img width="1019" alt="Screenshot 2022-08-04 at 14 10 16" src="https://user-images.githubusercontent.com/107257153/182855641-a9d497ab-2913-4850-b7f1-438b3a53a714.png">
<img width="1024" alt="Screenshot 2022-08-04 at 14 10 23" src="https://user-images.githubusercontent.com/107257153/182855701-a6175b6c-1073-49fd-a0be-f181e5e24664.png">

**Comment by ivanromakh** [comment]
@s-steddy I am not sure what you mean. in my chrome it is just recomnending passwords but not filling them.
![image](https://user-images.githubusercontent.com/18567511/184143980-fbd7d5dc-562f-49ce-9f44-5f70830c1dd3.png)

**Comment by s-steddy** [comment]
This looks to have been fixed for me. Thank you

**Comment by ivanromakh** [comment]
browser always trying to put password and email somewhere in fields.

**Comment by s-steddy** [comment]
Yep, looks to be fixed in my firefox in dev (but not production). So all good for next release!

**Comment by s-steddy** [comment]
Ok it does need one last fix, sorry:

- Obscure PIN numbers as they are entered:
<img width="200" alt="Screenshot 2022-08-16 at 15 56 56" src="https://user-images.githubusercontent.com/107257153/184912087-dbcc99f9-510a-41e9-a808-da519067c5b8.png">

- And once account is connected, as they are stored:
<img width="200" alt="Screenshot 2022-08-16 at 15 55 27" src="https://user-images.githubusercontent.com/107257153/184912104-bcc79ba2-2665-4e3c-9cdb-c9fd3cf09873.png">

- If you click unlink, remove numbers from barcode and pin fields, and set selection box back to 'Library Authority'

**Comment by s-steddy** [comment]
With a connected account, the Pin field should read ••••, or •••••• depending on number of characters, instead of 'Pin', otherwise it looks like a text entry field.

<img width="500" alt="Screenshot 2022-08-26 at 13 08 32" src="https://user-images.githubusercontent.com/107257153/186900293-a9ae25c8-6792-4369-b831-51394b5d06b7.png">

Can anything be done about the line underneath the dropdown menu for authority?

**Comment by ivanromakh** [comment]
@s-steddy should I remove line?

**Comment by s-steddy** [comment]
If possible, unless there's a reason for it I was unaware of?

Actually once the account is linked, what woudl be ideal would be to remove arrow in the dropdown menu, and to fade the text in all three fields, to indicate they can't be changed (until you click unlink account)

**Comment by ivanromakh** [comment]
nope it is easy
